src/types/meta-assistant/index.ts

```diff
@@ -17,18 +17,16 @@ export default {
  * DailyCombinedData interface
  */
 export interface DailyCombinedData {
```

Comment:
This is ok but imports are not correct, could you scroll up and fix if 
import type { ChatPostMessageResponse } from "@slack/web-api/dist/response/ChatPostMessageResponse";
import type { DateTime } from "luxon";
import type { PersonTotalTime } from "src/generated/client/api";